uisdk/src/main/java/com/karhoo/uisdk/screen/booking/map/BookingMapView.kt

```diff
@@ -54,10 +54,7 @@ import com.karhoo.uisdk.util.extension.isLocateMeEnabled
 import com.karhoo.uisdk.util.extension.orZero
 import com.karhoo.uisdk.util.extension.showCurvedPolyline
 import com.karhoo.uisdk.util.extension.showShadowedPolyLine
-import kotlinx.android.synthetic.main.uisdk_view_booking_map.view.bookingMapLayout
-import kotlinx.android.synthetic.main.uisdk_view_booking_map.view.locateMeButton
-import kotlinx.android.synthetic.main.uisdk_view_booking_map.view.mapView
-import kotlinx.android.synthetic.main.uisdk_view_booking_map.view.pickupPinIcon
+import kotlinx.android.synthetic.main.uisdk_view_booking_map.view.*
```

Comment:
can we avoid having wildcard imports 
If you import Karhoo codeStyle, it can help